lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -489,16 +490,27 @@ def test_ccx_invite_enroll_up_to_limit(self):
         }
         response = self.client.post(url, data=data, follow=True)
         self.assertEqual(response.status_code, 200)
-        # a CcxMembership exists for the first two students but not the third
+        # even if course is coach can enroll staff and admins of master course into ccx
         self.assertTrue(
-            CourseEnrollment.objects.filter(course_id=ccx_course_key, user=students[0]).exists()
+            CourseEnrollment.objects.filter(course_id=ccx_course_key, user=self.staff).exists()
         )
         self.assertTrue(
-            CourseEnrollment.objects.filter(course_id=ccx_course_key, user=students[1]).exists()
+            CourseEnrollment.objects.filter(course_id=ccx_course_key, user=self.instructor).exists()
         )
-        self.assertFalse(
+        self.assertTrue(
+            CourseEnrollment.objects.filter(course_id=ccx_course_key, user=self.coach).exists()
+        )
+
+        # a CcxMembership exists for the first five students but not the sixth
+        self.assertTrue(
             CourseEnrollment.objects.filter(course_id=ccx_course_key, user=students[2]).exists()
```

Comment:
`students[2]` overlaps with `self.coach`, I think you need to add one to each index here so you're starting at `students[3]` instead